Automattic/jetpack
# Remove Stats copy from connection screen

See related issue

## Proposed changes:

* This PR simply removes the copy related to Stats from the connection screen.

| Before | After |
| ------ |  -----|
| <img width="1136" alt="image" src="https://github.com/user-attachments/assets/b5af0dab-05d5-4393-a107-925d6a71bfae"> | <img width="1139" alt="image" src="https://github.com/user-attachments/assets/55425acf-9774-47e2-a8ed-4428d148c707"> |


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Spin up a JN site with Jetpack Beta pointing to this branch
* Visit `/wp-admin/admin.php?page=jetpack#/` and confirm there's no mention to "Stats" in the features list
